src/backend/commands/event_trigger.c

```diff
@@ -1895,7 +1896,20 @@ pg_event_trigger_ddl_commands(PG_FUNCTION_ARGS)
 					 * rather than failing.  This can happen for example with
 					 * some subcommand combinations of ALTER TABLE.
 					 */
-					identity = getObjectIdentity(&addr, true);
+
+					/*
+					 * When PgAudit extension is enabled we want to initialise 
+					 * the syscache lookup for babelfish temp tables and table 
+					 * variable from postgres dialect.
+					 */
```

Comment:
comment should be in hooks.c in extension

Reply:
+1